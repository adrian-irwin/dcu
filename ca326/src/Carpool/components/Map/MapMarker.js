import React from "react";
import { Marker } from "react-native-maps";

const MapMarker = ({ long, lat, title, desc, colour, id }) => {
    return (
        <Marker
            coordinate={{ latitude: lat, longitude: long }}
            title={title}
            description={desc}
            pinColor={colour}
            identifier={id}
        />
    );
};

export default MapMarker;
