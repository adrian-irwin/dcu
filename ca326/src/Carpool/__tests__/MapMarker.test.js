import React from "react";
import { render } from "@testing-library/react-native";

import MapMarker from "../components/Map/MapMarker";

describe("MapMarker", () => {
    it("renders correctly", () => {
        const { toJSON } = render(<MapMarker />);
        expect(toJSON()).toMatchSnapshot();
    });
});
