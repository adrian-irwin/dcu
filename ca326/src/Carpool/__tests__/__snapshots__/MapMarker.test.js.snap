// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MapMarker renders correctly 1`] = `
<AIRMapMarker
  coordinate={
    Object {
      "latitude": undefined,
      "longitude": undefined,
    }
  }
  onPress={[Function]}
  style={
    Array [
      Object {
        "backgroundColor": "transparent",
        "position": "absolute",
      },
      undefined,
    ]
  }
/>
`;
